src/explorer/DeploymentSlotTreeItem.ts

```diff
@@ -3,15 +3,24 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { SiteConfig } from 'azure-arm-website/lib/models';
 import * as path from 'path';
-import { AppSettingsTreeItem, ISiteTreeRoot, SiteClient } from 'vscode-azureappservice';
+import { AppSettingsTreeItem, DeploymentsTreeItem, ISiteTreeRoot, SiteClient } from 'vscode-azureappservice';
 import { AzureParentTreeItem, AzureTreeItem } from 'vscode-azureextensionui';
 import { FolderTreeItem } from './FolderTreeItem';
 import { SiteTreeItem } from './SiteTreeItem';
 
 export class DeploymentSlotTreeItem extends SiteTreeItem {
+
+    public get iconPath(): { light: string, dark: string } {

```

Comment:
nit: Why'd you move iconPath to the top of the class? I like it better where it was

Reply:
I think it got automatically moved on save.